### PR TITLE
added stacks and queues

### PR DESCRIPTION
**Pull request by daneng1**
- [x]   Top-level README “Table of Contents” is updated
 - [x]   Feature tasks for this challenge are completed
 - [x]   Unit tests written and passing
 - [x]   “Happy Path” - Expected outcome
- [x]    Expected failure
- [x]    Edge Case (if applicable/obvious)
 - [x]   README for this challenge is complete
- [x]    Summary, Description, Approach & Efficiency, Solution
 - [x]   Link to code
 - [x]   Picture of whiteboard

### CODE REVIEW

File: challenges/stacks-queues/stack.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const Node = require('./node.js');
+
+function Stack() {
```
Comment (lizkavalski) [inline challenges/stacks-queues/stack.js:5]: Should be a Class
